scopes/workspace/install/install.main.runtime.ts

```diff
@@ -785,7 +785,7 @@ export class InstallMain {
       if (component.state._consumer.removed) return;
       const pkgName = componentIdToPackageName(component.state._consumer);
       const found = workspacePolicy.find(pkgName);
-      if (found) {
+      if (found && found.lifecycleType !== 'peer') {
```

Comment:
I wonder about this, do we want to allow a peer dep to be both in policy and bitmap?
why treat here peers differently?

Reply:
Aren't peers allowed in policy right now?

Reply:
This change was added to fix this issue:

```
     > my-scope/ui/button ...  issues found
       tracked component added as a package  (either remove the package from the workspace.jsonc (bit uninstall) or remove the component (bit remove)):
          @my-scope/ui.button

```

Reply:
when you got this error do you had it in both the policy and the bitmap?

Reply:
no, only in policy. Not in bitmap.

Reply:
Isn't that checking for it in the policy? It is in the policy. It isn't in bitmap.

Reply:
I misunderstood you at first. I thought you meant that the peer is also added via `bit deps set`. Yes, the peer is in the workspace, so it is in the bitmap as a component.

So what you suggest? Not supporting it via policy if the component is in the workspace?